explainer.md

```diff
@@ -57,7 +57,7 @@ On platforms that only use file extensions to describe file types, user agents c
 
 The user can right click on CSV or SVG files in the operating system's file browser, and choose to open the files with the Grafr web application. (This option would only be presented if Grafr has been [installed](https://w3c.github.io/manifest/#installable-web-applications).).
 
-Choosing to open the file would create a new top level browsing context, navigating to '{origin}{action}'. Assuming the user opened `graph.csv` in Grafr the URL would be `https://grafr.com/open-csv/`.
+Choosing to open the file would create a new top level browsing context, navigating to the `action` URL (resolved against the manifest URL). Assuming the user opened `graph.csv` in Grafr the URL would be `https://grafr.com/open-csv/`.
```

Comment:
Manifest parent URL?

Reply:
+1

Reply:
No, it is resolved against the manifest URL itself. The resolution process automatically strips off the last path segment. So it would be incorrect to say it is resolved against the "manifest parent URL".

See https://www.w3.org/TR/appmanifest/#scope-member for what this would look like:

> Let scope URL be the result of parsing value, using manifest URL as the base URL.